docs/config.c

```diff
@@ -567,6 +567,12 @@
 ** information on how to set $$compose_format.
 */
 
+{ "compose_show_user_headers", DT_BOOL, true },
+/*
+** .pp
+** When \fIset\fP, Neomutt will display custom headers on the compose screen.
```

Comment:
Can you add a sentence mentioning the `my_hdr` command.
If you quote it as `$my_hdr` it will be auto-linked in the manual.

Reply:
Added with 179d074. Also here used 'user-defined headers' over 'custom headers' (the former is more common in existing docs)